rust/gitxetcore/src/xetblob/xet_repo.rs

```diff
@@ -224,7 +224,7 @@ impl XetRepo {
         let remote = if let Some(repo_info) = repo_info.as_ref() {
             repo_info.repo.html_url.clone()
```

Comment:
I think we need to pass this url into config so that `gitxetcore::data::cas_interface::create_cas_client` can pick it up and use it for billing purpose

Reply:
With discussion offline -- it's likely this is the case here, although user and auth information is populated in the config struct.   We need to clean that up for sure, but we'll get this PR in here and do that in the next one.